modules/core/src/main/scala/org/scalasteward/core/repoconfig/RepoConfig.scala

```diff
@@ -31,7 +31,7 @@ final case class RepoConfig(
 }
 
 object RepoConfig {
-  val default: RepoConfig = RepoConfig()
+  private[core] val default: RepoConfig = RepoConfig()
```

Comment:
What is the purpose of this change? Is it because we now have two defaults, `RepConfig.default` and `RepoConfigAlg.defaultRepoConfig`, and that is confusing? Would it be less confusing if this is renamed to `RepoConfig.empty`?

Reply:
Just changed visibility, actually I want it to be used only by `RepoConfigAlg`, so I'll make it `private[repoconfig]`
`RepoConfig.empty` makes more sense for me, will rename